### PR TITLE
Add new board and advisory board members

### PR DESCRIPTION
**Pull request by MorganHaworth**
Please verify updated members and their roles. Let me know what changes can be made.
Removed: 
- Grace: Vice President
- Lauren P: Secretary
- Mary Clare Rogers: Treasurer
- Melinda
- Autumn: Board Member
- Bianca
- Trisha
- Ogeoma
- Stefanie
- Jessica S

Added: 
- Autumn: Vice President (Organizer)
- Grace: Treasurer (Instructor, Teaching Assistant, Organizer)
- Mary Clare Rogers: Board Member (Organizer)
- Mary Donovan Martello: Board Member (Teaching Assistant, Organizer)
- Morgan Haworth: Board Member (Instructor)
- Tera Maher: Board Member (Instructor)
- Lauren Schultz: Advisory Board (Organizer)

### CODE REVIEW

File: pages/volunteer.liquid [hunk 1]
```diff
@@ -123,37 +123,35 @@ eleventyNavigation:
 
       <div>
         <div class="bio-with-pic">
-          <img class="bio-pic" src="/assets/images/volunteers/wendy-pic.jpg" alt="Wendy Holley headshot">
-          <h3>Wendy Holley</h3>
+          <img class="bio-pic" src="/assets/images/volunteers/MorganHaworth.jpg" alt="Morgan Haworth headshot">
+          <h3>Morgan Haworth</h3>
           <h4>Board Member</h4>
           <div class="badge-container">
             <div class="gwc-badge instructor">{I}</div>
```
Comment (Gmfholley) [inline pages/volunteer.liquid:130]: We should add that you're an organizer as well, Morgan!

File: pages/volunteer.liquid [hunk 2]
```diff
@@ -123,37 +123,35 @@ eleventyNavigation:
...
-          <img class="bio-pic" src="/assets/images/volunteers/biancaz.jpg" alt="Bianca Zongrone Jefferson headshot">
-          <h3>Bianca Zongrone Jefferson</h3>
+          <img class="bio-pic" src="/assets/images/volunteers/tera.jpg" alt="Tera Maher headshot">
+          <h3>Tera Maher</h3>
           <h4>Board Member</h4>
           <div class="badge-container">
-            <div class="gwc-badge ta">[TA]</div>
-            <div class="gwc-badge speaker">"S"</div>
-            <div class="gwc-badge organizer">&lt;O&gt;</div>
+            <div class="gwc-badge instructor">{I}</div>
```
Comment (Gmfholley) [inline pages/volunteer.liquid:154]: I think Tera has been a speaker rather than instructor?  I'm happy to be corrected.

**Review by Gmfholley — APPROVED** [review]
Looks great!  The preview looks 💯 I left a few comments but I am happy to be corrected.  If I am wrong about those things, this is good to merge.

File: pages/volunteer.liquid (hunk 2 above)
Comment (MorganHaworth) [reply pages/volunteer.liquid:154]: I thought she lead a Code Crush thing one day, or is that a speaker?

**Pull request merged**